TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -476,7 +476,7 @@ <h4 class="timeline-title">{% trans "Collections" %}</h4>
           {% else %}
             <div class="panel-body top-border">
               {% if user.is_authenticated %}
-                {% if user.editor.wp_bundle_eligible %}
+                {% if user.editor.wp_bundle_eligible and user.userprofile.terms_of_use %}
```

Comment:
Shouldn't need this since we're doing the terms check as part of the bundle eligibility check

Reply:
Done! I removed it and it seems to be working once I added the bundle validation when accepting or disagreeing with the terms.